debezium-core/src/main/java/io/debezium/transforms/TimezoneConverter.java

```diff
@@ -307,12 +308,38 @@ private void handleStructs(Struct value, Type type, String matchName, Set<String
 
         Struct before = getStruct(value, Envelope.FieldName.BEFORE);
         Struct after = getStruct(value, Envelope.FieldName.AFTER);
+        Struct source = getStruct(value, Envelope.FieldName.SOURCE);
+
+        Set<String> beforeFields = new HashSet<>();
+        Set<String> afterFields = new HashSet<>();
+        Set<String> sourceFields = new HashSet<>();
+
+        if (!fields.isEmpty() && !fields.contains(null)) {
```

Comment:
In what use case would the `Set<String>` fields contain a `null` entry rather than simply being empty?

Reply:
@Naros Sorry I missed this comment. This check is added from here while parsing each of the include/exclude items. https://github.com/debezium/debezium/blob/c2eac1714eaa560817a00e83ba215dce1a0e9bca/debezium-core/src/main/java/io/debezium/transforms/TimezoneConverter.java#L400

I have added proper null check for that.